src/views/Dashboard/components/Widgets/SendWidget/SendWidget.tsx

```diff
@@ -88,15 +88,15 @@ const StyledWidgetHeading = styled(Typography)(() => ({
 
 const StyledToAddressInput = styled(JUPInput, {
   shouldForwardProp: (prop) => prop !== "onChange",
-})<{ onChange?: (e: any) => void }>(({ onChange }) => ({
+})<{ onChange?: (e: React.SyntheticEvent) => void }>(() => ({
```

Comment:
create an issue for specifically discussing this one with V, I wonder if passing `{...other}` through to the component would fix this in the styling

Reply:
https://github.com/Up-North-Coding/jovian/issues/100